DOCS/man/input.rst

```diff
@@ -1961,6 +1937,44 @@ Property list
     whether the video window is visible. If the ``--force-window`` option is
     used, this is usually always returns ``yes``.
 
+``vo-performance``
+    Some video output performance metrics. Not implemented by all VOs. This has
+    a number of sup-properties, of the form ``vo-performance/<metric>-<value>``,
+    all of them in milliseconds.
```

Comment:
I think this should say microseconds. I get values in the hundreds and thousands (and my video is smooth).